resources/styleheets/layout/_invi.scss

```
@@ -0,0 +1,37 @@
+
+main {
+    margin-top: $p6;
+    flex: 57%;
+    margin-left: 13%;
+    text-align: $ali_center;
+
+    @media (max-width: 800px) {
+        .centered-box {
+            max-width: 90%; 
+        }
+
+        flex: 1;
+        margin-left: 0;
+
+
+    }
```

```suggestion
  @media (max-width: 800px) {
    flex: 1;
    margin-left: 0;
  
    .centered-box {
      max-width: 90%
    }
  }
```
first the style for the selected element then the nested element